app/assets/stylesheets/components/_images.scss

```diff
@@ -39,6 +39,16 @@ figure {
     }
   }
 
+  // BROKEN 
```

Comment:
Could someone have a look? Sandy wants an image + caption centered at the bottom of the news page.